src/test/resources/features/campaign.feature

```diff
@@ -2,19 +2,27 @@ Feature: Requests for Campaign endpoint
 
   @GetCampaigns
   Scenario: Get all Campaigns
-    Given
+  Given
     When I add this "/Campaign" endpoint and send the request
     Then the response status code should be "200"
 
   @CreateCampaign
   Scenario Outline: Create a Campaign
     Given I build the pathParams and body to request
-      | name  | <nameProduct> |
+      | name     | <nameCampaign>     |
+      | isActive | <isActiveCampaign> |
+      | type     | <typeCampaign>     |
+      | status   | <statusCampaign>   |
     When I add this "/Campaign" endpoint and send the request with body
```

Comment:
Could be "I set the endpoint to "/Campaign"  and execute the request with the body"